config_defaults.yml

```diff
@@ -71,7 +71,7 @@
     dist: trusty
     services: docker
     bundler_args: --with system_tests
-    env: PUPPET_INSTALL_TYPE=agent BEAKER_debug=true BEAKER_PUPPET_COLLECTION=@@COLLECTION@@ BEAKER_set=@@SET@@ BEAKER_TESTMODE=@@TESTMODE@@
+    env: PUPPET_INSTALL_TYPE=agent BEAKER_debug=true BEAKER_PUPPET_COLLECTION=@@COLLECTION@@ BEAKER_set=@@SET@@ BEAKER_setfile=@@SETFILE@@ BEAKER_TESTMODE=@@TESTMODE@@
```

Comment:
beaker-rspec does the right thing here:

https://github.com/puppetlabs/beaker-rspec/blob/da9259ffab8fdb248b3fa5c2e808f94a3b71625a/lib/beaker-rspec/spec_helper.rb#L41

Reply:
@DavidS what am I supposed to set in my `.sync.yml` then?
I tried with:
```yaml
.travis.yml:
  docker_sets:
    - set: 'centos7-64{hypervisor=docker}'
```
But it's looking for a file named `spec/acceptance/nodeset/centos7-64{hypervisor=docker}.yaml`